scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/web/DatasetSuccessController.java

```diff
@@ -0,0 +1,15 @@
+package edu.asu.diging.scriptoocloud.web;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+@Controller
+public class DatasetSuccessController {
+
+
+    @GetMapping("/datasets/success")
+    public String get(Model model) {
```

Comment:
after creation of a dataset, the user should either see the list of datasets or the newly created dataset